Source/VMware.vSphereDSC/VMware.vSphereDSC.psm1

```diff
@@ -16,1335 +16,1451 @@ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
 
 Using module '.\VMware.vSphereDSC.Helper.psm1'
 
-enum Ensure
-{
-    Absent
-    Present
+enum Ensure {
+  Absent
+  Present
 }
 
-enum ServicePolicy
-{
-    Unset
-    On
-    Off
-    Automatic
+enum ServicePolicy {
+  Unset
+  On
+  Off
+  Automatic
 }
 
-enum Period
-{
-   Day = 86400
-   Week = 604800
-   Month = 2629800
-   Year = 31556926
+enum Period {
+  Day = 86400
+  Week = 604800
+  Month = 2629800
+  Year = 31556926
 }
 
-enum LoggingLevel
-{
-    Unset 
-    None
-    Error
-    Warning
-    Info
-    Verbose
-    Trivia
+enum LoggingLevel {
+  Unset
+  None
+  Error
+  Warning
+  Info
+  Verbose
+  Trivia
 }
 
-class BaseDSC
-{
-    <#
+class BaseDSC {
+  <#
     .DESCRIPTION
 
     Name of the Server we are trying to connect to. The Server can be a vCenter or ESXi.
     #>
-    [DscProperty(Key)]
-    [string] $Server
+  [DscProperty(Key)]
+  [string] $Server
 
-    <#
+  <#
     .DESCRIPTION
 
     Credentials needed for connection to the specified Server.
     #>
-    [DscProperty(Mandatory)]
-    [PSCredential] $Credential
+  [DscProperty(Mandatory)]
+  [PSCredential] $Credential
 
-    <#
+  <#
     .DESCRIPTION
 
     Established connection to the specified vSphere Server.
     #>
-    hidden [PSObject] $Connection
+  hidden [PSObject] $Connection
 
-    <#
+  <#
     .DESCRIPTION
 
     Imports the needed VMware Modules.
     #>
-    [void] ImportRequiredModules()
-    {
-	    $savedVerbosePreference = $global:VerbosePreference
-        $global:VerbosePreference = 'SilentlyContinue'
-		
-        Import-Module -Name VMware.VimAutomation.Core
-		
-		$global:VerbosePreference = $savedVerbosePreference
-    }
+  [void] ImportRequiredModules() {
+    $savedVerbosePreference = $global:VerbosePreference
+    $global:VerbosePreference = 'SilentlyContinue'
 
-    <#
+    #        Set-PowerCLIConfiguration -ParticipateInCeip:$false -DisplayDeprecationWarnings:$false -Scope Session -Confirm:$false
+    Import-Module -Name VMware.VimAutomation.Core
+
+    $global:VerbosePreference = $savedVerbosePreference
+  }
+
+  <#
     .DESCRIPTION
 
-    Connects to the specified Server with the passed Credentials. 
+    Connects to the specified Server with the passed Credentials.
     The method sets the Connection property to the established connection.
     If connection cannot be established, the method writes an error.
     #>
-    [void] ConnectVIServer()
-    {
-        $this.ImportRequiredModules()
+  [void] ConnectVIServer() {
```

Comment:
Below there are unneeded indent and code format changes. We probably have to set standards coding standards. We'll address this in the coding guidelines later.

This makes the review of the actual changes a bit difficult. Is it possible to revert indent and code reformat changes?

Reply:
These changes in indentation and formatting came from the editor (VSC) I used and my global settings in that editor.
Would it be possible to add a Workspace settings file for VSC to the repo, with your desired formatting?

Reply:
To change the files for now, what are you current rules?
- tab is 4 spaces
- code block starts on a new line

Any other rules that are apparently different between your side and mine?

Reply:
The .psm1 file should now only show you the changes I made, no more formatting changes.
Can you check the diff?

Reply:
@SimeonGerginov started working on VSC formatting file to avoid such diffs.

To be able to merge your PR the commits have to be signed. You can use your email to sign commits. If you set your user.name and user.email git config values, you can sign your commit automatically with git commit -s.

git config --global user.name <user>
git config --global user.email <email>